src/core/maprenderer/qgsmaprendererjob.cpp

```diff
@@ -644,19 +700,24 @@ LayerRenderJobs QgsMapRendererJob::prepareSecondPassJobs( LayerRenderJobs &first
     const QSet<QgsSymbolLayerId> &symbolList = it->first;
 
     // copy the initial job ...
-    secondPassJobs.append( LayerRenderJob() );
-    LayerRenderJob &job2 = secondPassJobs.last();
+    secondPassJobs.emplace_back( LayerRenderJob() );
+    LayerRenderJob &job2 = secondPassJobs.back();
+
+#if 0
```

Comment:
isn't some initialization of job2 lacking now that job2 = job is disabled ?

Reply:
Oops, had disabled this temporarily..